src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/Document.java

```diff
@@ -23,12 +23,6 @@
     @JsonProperty("id")
     private String id;
 
-    @JsonProperty("url")
```

Comment:
Will we not be using these fields in future? e.g. url/name etc

Reply:
There is no need as of now, can be added later if need be.